recipes/ffmpeg/bld.bat

```diff
@@ -0,0 +1,3 @@
+:: http://ffmpeg.zeranoe.com/blog/?p=300
+
+bash "%RECIPE_DIR%\mingw-w64-build-3.6.0.sh"
```

Comment:
It looks like this code from the version in conda-recipes repo, which indeed does nothing to build ffmpeg for Windows. See [this discussion and related issues at PyAV](https://github.com/mikeboers/PyAV/pull/84), where one user shared instructions for building FFmpeg on Windows (which is quite an ordeal).